com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -455,6 +455,12 @@ void OnPreCull()
             if (m_Camera.stereoEnabled)
             {
                 m_Camera.ResetStereoProjectionMatrices();
+                m_Camera.CopyStereoDeviceProjectionMatrixToNonJittered(Camera.StereoscopicEye.Left); // device to unjittered
+                m_Camera.CopyStereoDeviceProjectionMatrixToNonJittered(Camera.StereoscopicEye.Right);
+                m_Camera.projectionMatrix = m_Camera.GetStereoNonJitteredProjectionMatrix(Camera.StereoscopicEye.Left);
+                m_Camera.nonJitteredProjectionMatrix = m_Camera.projectionMatrix;
+                m_Camera.SetStereoProjectionMatrix(Camera.StereoscopicEye.Left, m_Camera.GetStereoProjectionMatrix(Camera.StereoscopicEye.Left));
+                m_Camera.SetStereoProjectionMatrix(Camera.StereoscopicEye.Right, m_Camera.GetStereoProjectionMatrix(Camera.StereoscopicEye.Right));
```

Comment:
The reason for the eye shift is m_Camera.ResetProjectionMatrix(); on line 450 resets the camera projection matrix to something that isn't the same as the eye's projection matrix.

Basically, the camera's renderstereo() sets the stereo projection matrix for each eye here:
https://sourcegraph.ds.unity3d.com/github.cds.internal.unity3d.com/unity/unity/-/blob/Runtime/Camera/Camera.cpp?L3468
which runs before precull.

Then in the precull of the postprocessinglayer and if we have TAA enabled... we reset the projection matrix of the camera which is the data that seems to trickle down somewhere into TAA even though we eventually have our stereo projection matrices set later on but i've been logging who’s using those matrices and it was getting pulled from multiplace places. (I didn’t dig all the way down to see the extent)

So from my perspective, we have 2 choices. 
Either we don't reset the projection matrix in the precull and everything seems fine but i spoke with Thomas Hourdel who wrote the reset projection matrix bit and he stood by his reasons to do so but we could essentially not do it while in stereo. (not 100% sure it fixes all issues)
If postprocess must reset the projection matrix in the precull… let’s make sure that we reset it for everything to the same thing and using the device sounded like the most trustworthy location to get it from but granted… there might be some overhead to it.

Reply:
something to also note is that m_Camera.SetStereoProjectionMatrix doens't do anything in non multipass
because of:
`    if (stereoRenderingPath != kStereoRenderingMultiPass)
    {
        WarningStringObject("Can't set custom eye projection matrix when not in multipass mode", this);
        return;
    }`

and constantly spits out warnings.

Reply:
`Currently, this general fix of copying and setting of stereo matrices adds cost for all effects (including non-TAA effects) and adds unnecessary cost for stereo instancing as well.`
I can put this change behind an if statement for TAA, multipass or do you think there's a better fix?

Reply:
hopefully this is the last comment about this. :)
the original bug complains about a shift in the right eye but there's also a shift in the left eye if you compare it against no AA.